pyigm/cgm/analysis.py

```diff
@@ -0,0 +1,61 @@
+""" module for analysis of CGM outside the CLasses
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+
+import numpy as np
+import warnings
+import pdb
+
+from scipy.special import gamma, gammainc
+
+from astropy import units as u
+from astropy import constants as const
+from astropy.cosmology import Planck15
+
+def dndx_rvir(Lrng=(0.001, 10), beta=0.2, rvir_Lstar=250.*u.kpc,
+         phi_str = 1.49/u.Mpc**3, alpha = -1.05, Mstar = -20.44,
+         cosmo=None):  #  ; M* - 5 log h
+    """ Estimate dN/dX for a set of CGM assuming unit covering fraction
+    Use beta=0 and rvir_Lstar=300kpc for a constant CGM to 300kpc
+
+    Parameters
+    ----------
+    Lrng : tuple, optional
+      Range to integrate luminosity in terms of L*
+    beta : float, optional
+      Parameterization of rvir with L
+       r_vir = 250 kpc * (L/L*)^beta
+    phi_str, alpha, Mstar : Quantity, float, float
```

Comment:
Should indicate somewhere that phi_str has a factor of h<sup>-3</sup> in it as well.